g/gotest.tools/gotest.tools_ubi_8.4.sh

```diff
@@ -2,9 +2,9 @@
 # ----------------------------------------------------------------------------
 #
 # Package       : gotest.tools
-# Version       : v3.0.2, v3.0.3
+# Version       : v3.4.0
```

Comment:
@Chandranana  we should not remove the old versions, just add the new version instead and make sure old versions work on UBI 8.5 as well

Reply:
Done.